### PR TITLE
Make CMDLINE overridable from outside

### PR DESCRIPTION
**Pull request by LocutusOfBorg**
currently there is no way to override the default CMDLINE, lets set if only if not already set from outside

### CODE REVIEW

**Comment by LocutusOfBorg** [comment]
Can you please also fix for morty and other branches? :)

**Comment by mirzak** [comment]
One could actually override with with an .bbappend file, but I support this change an was actually thinking about sending a patch as well. 

One problem with this, patches should go to the mailing list. See README file in meta-raspberrypi

**Comment by LocutusOfBorg** [comment]
I already have a bbappend, but the override must have ":=" (I think) and this seems somewhat strange

**Comment by LocutusOfBorg** [comment]
thanks I think I forwarded it to the ML!

**Comment by betafive** [comment]
@agherzan Just spotted this on the mailing list, I think we can close this pull request.

File: recipes-kernel/linux/linux-raspberrypi.inc [hunk 1]
```diff
@@ -21,7 +21,7 @@ KERNEL_DEFCONFIG_raspberrypi2 ?= "bcm2709_defconfig"
 KERNEL_DEFCONFIG_raspberrypi3-64 ?= "bcmrpi3_defconfig"
```
Comment (agherzan) [inline recipes-kernel/linux/linux-raspberrypi.inc]: Can you please modify the commit log in accordance to http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines ? Looks good to me other than that.

Comment (LocutusOfBorg) [inline recipes-kernel/linux/linux-raspberrypi.inc]: I think it is good now :)

**Comment by agherzan** [comment]
With one exception : the subject line. It needs to be in the following format : "component : Short description". 

Thanks once again for your contribution.

**Comment by LocutusOfBorg** [comment]
I did correct that subject line and added a new commit on raspberrypi0.conf :)

**Comment by agherzan** [comment]
Hi @LocutusOfBorg I'm really sorry for buzzing you again. But I think once you get it, everything will be just fine.

The first commit makes a change in the linux-raspberrypi component. The commit subject therefore should have a format like: "linux-raspberrypi: Make CMDLINE overridable from outside".

Similar for the second commit where the component is a machine configuration: "raspberrypi0.conf: make SERIAL_CONSOLE overridable from outside"

Take a look [here](http://www.openembedded.org/wiki/Commit_Patch_Message_Guidelines) for additional information.

**Comment by agherzan** [comment]
ok to test

**Comment by LocutusOfBorg** [comment]
thanks! :) I guess I fixed it correctly

**Comment by agherzan** [comment]
@LocutusOfBorg looks good now. Thanks for this.

**Comment by LocutusOfBorg** [comment]
thanks for teaching me how to fix the commit messages :)

**Comment by agherzan** [comment]
With pleasure. Merging this.

**Pull request merged**